admin/database/database.go

```diff
@@ -212,8 +223,8 @@ type InsertProjectOptions struct {
 	ProductionOLAPDSN    string
 	ProductionSlots      int
 	ProductionBranch     string
-	GithubURL            *string
-	GithubInstallationID *int64
+	GithubURL            *string `validate:"omitempty,http_url"`
```

Comment:
Any particular reason for adding `omitempty` on `GithubURL`. As I understand this can't be kept blank in current flow.

Reply:
`GithubURL` is nullable in the DB, so this seemed safest

Reply:
May be we should introduce non nullable clause in the DB as well. I see no reason for a project to have empty/null github URL.

Reply:
Agree – took a look and I think it's because the migration had to be backwards compatible. Adding an item to flatten migrations and tighten constraints before pushing to prod.